examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+  "port": 27017,
+  "containerPort": 27017,
+  "labels": {
+    "name": "meanio-mongodb"
+  },
+  "selector": {
+    "name": "meanio-mongodb"
+  }
+}
+```
+
+to create the service by running:
+
+```shell
+$ kubectl create -f examples/meanio/meanio-mongodb-service.json
+meanio-mongodb
+
+$ kubectl get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+meanio-mongodb          name=meanio-mongodb                                             name=meanio-mongodb          10.0.222.168        27017
+```
+
+This will cause all pods to see the MongoDB server apparently running on <ip>:27017.
+
+- Traffic will be forwarded from the service "port" (on the service node) to the  *containerPort* on the pod which (a node the service listens to). 
+
+Thus, once created, the service proxy on each minion is configured to set up a proxy on the specified port (in this case port 27017).
+
+
+make sure that the mongodb pod is running and listening by running:
+
+```shell
+
+$ kubectl log meanio-mongodb-controller-nyxxv
+
+```
+
+### Step Three: Create the meanio pod
+
+This is a nodejs server that is configured to talk to the MognoDB server. 
+Note that you will need the MongoDB server up and running before running the meanio pod.
+
+The pod is described in the file `examples/meanio/meanio-controller.json`:
+
+```js
+{
+  "id": "meanio-controller",
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta1",
+  "desiredState": {
+    "replicas": 1,
+    "replicaSelector": {"name": "meanio"},
+    "podTemplate": {
+      "desiredState": {
+         "manifest": {
+           "version": "v1beta1",
+           "id": "meanio",
+           "containers": [{
+             "name": "meanio",
+             "image":"shaiweinstein/k8s:v1",
+             "cpu": 100,
+             "memory": 2000000000,
+             "ports": [{"name": "meanio-server", "containerPort": 3000}],
+             "env": [{"name": "DB_PORT_27017_TCP_ADDR", "value": "meanio-mongodb" }]
+           }]
+         }
+       },
+       "labels": {
+         "name": "meanio",
+         "uses": "meanio-mongodb",
+         "app": "meanio"
+       }
+      }},
+  "labels": {"name": "meanio"}
+}
+```
+
+Using this file, you can turn up your meanio framework  with:
+
+```shell
+$ kubectl create -f examples/meanio/meanio-controller.json
+meanio-controller
+
+$ kubectl get rc
+CONTROLLER                             CONTAINER(S)            IMAGE(S)                                   SELECTOR                     REPLICAS
+meanio-controller                      meanio                  shaiweinstein/k8s:v1                name=meanio                  1
+meanio-mongodb-controller              meanio-mongodb          mongo:2.6.8                         name=meanio-mongodb          1
+
+```
+
+Once that's up (it may take a few minutes to create the pod) you can list the pods, to verify that the mongodb and meanio are running:
```

Comment:
I think you might mean "... the mongodb and meanio **pods** are running:"

Also, is there a better way to describe the second pod, instead of "meanio"? Maybe a "frontend" pod?  I think of "MEAN" as the entire stack, so it's a bit confusing.